moodlecli/moodle.py

```diff
@@ -223,3 +225,6 @@ def get_users_by_course(self, course_id):
 
     def get_course_enrolment_url(self, course_id):
         return f"{self.moodle_url}/enrol/index.php?id={course_id}"
+
+    def get_user_uuids(self, user_ids):
```

Comment:
Maybe we can have `user_ids=[]` so this can be invoked as `.get_user_uuids()` for querying all versus a caller having to do `.get_user_uuids([])`.